src/Symfony/Component/Form/Form.php

```diff
@@ -359,7 +359,7 @@ public function setData($modelData)
             if (null !== $dataClass && !$viewData instanceof $dataClass) {
                 $actualType = \is_object($viewData)
                     ? 'an instance of class '.\get_class($viewData)
```

Comment:
looks like this can be removed and only use get_debug_type. in all other places we do not add the phrase `an instance of class` to the message.

Reply:
fixed